scripts/docbase.js

```diff
@@ -365,64 +367,73 @@
             getData: function() {
                 return new fetcher();
             },
-            getCommits: function(){
+            getCommits: function() {
                 var options = Docbase.options;
                 var file_path = $route.current.params;
-                var full_path = options.github.path+'/'+file_path.version+'/'+file_path.folder+'/'+file_path.file;
-                return $http.get('https://api.github.com/repos/'+ options.github.user +'/'+ options.github.repo+'/commits?path='+full_path+'.md');
+                var full_path = options.github.path + '/' + file_path.version + '/' + file_path.folder + '/' + file_path.file;
+                return $http.get('https://api.github.com/repos/' + options.github.user + '/' + options.github.repo + '/commits?path=' + full_path + '.md');
+            },
+            searchIndex:function(){
+                return $http.get('https://raw.githubusercontent.com/appbaseio/Docs/gh-pages/search-index.json');
```

Comment:
@mateusfreira Yes, plus the search functionality shouldn't be just here. It wouldn't work for the generated html files.